API.md

```diff
@@ -783,6 +783,38 @@ If an HTTP status code of 200 is returned, the body of the response will contain
 
 If there was an error, expect an HTTP status code in either the 4XX or 5XX range.
 
+### unregister
+
+Unregisters one replica of a data object.
+
+#### Request
+
+```bash
+curl http://localhost:<port>/irods-http-api/<version>/data-objects \
+    -H 'Authorization: Bearer <token>' \
+    --data-urlencode 'op=unregister' \
+    --data-urlencode 'lpath=<string>' \ # Absolute logical path to a data object.
+    --data-urlencode 'resource=<string>' \ # The resource holding the target replica. Optional.
+    --data-urlencode 'replica-number=<integer>' # The replica number of the target replica. Optional.
+```
+
+`resource` and `replica-number` are mutually exclusive parameters.
```

Comment:
The docs are incorrect. I'll fix that.

One of those params must be provided. I felt it made since for clients to be explicit rather than letting the system pick a random replica.

Reply:
agreed.

Reply:
Done